docs/tutorial/kymotracking.rst

```diff
@@ -484,6 +484,15 @@ by passing a width in pixels to sum counts over::
 
     tracks.save("tracks_signal.csv", sampling_width=3, correct_origin=True)
 
+We can re-import these tracks into Pylake at a later time using the :func:`~lumicks.pylake.load_tracks()` function::
+
+    tracks = lk.load_tracks("tracks.csv", kymo40, "green")
+
+.. note::
+
+    The `kymo` and `channel` arguments provided must be the same as the ones used to generate the tracks.
+    This includes any processing (e.g. cropping, downsampling, flipping) that was performed on the kymograph.
```

Comment:
I think Tiago pointed this out before, but I think we can be explicit that this is not an exhaustive list
```suggestion
    This includes any processing (e.g. cropping, downsampling, flipping, etc.) that was performed on the kymograph.
```